client/src/styles/global.scss

```diff
@@ -1,2 +1,31 @@
-@import "~@trussworks/react-uswds/lib/uswds.css";
 @import "~@trussworks/react-uswds/lib/index.css";
+@import "~@trussworks/react-uswds/lib/uswds.css";
+
+.j40FloatLogoLeft {
+  width: 100%;
+  padding-left: 50%;
+}
+
+.j40Footer {
+  font-family: "sans-serif";
+  font-size: 12px;
+  line-height: 24px;
+}
+
+.j40FooterAddressTitle {
+  line-height: 2.5;
```

Comment:
Likewise we should assume default line-height I think if possible, as this will vary based on font, language script, and device font settings

Reply:
The default size is 1.0 line height which isn't 508 compliant. We should investigate changing it globally?